pkg/config/config.go

```diff
@@ -516,6 +516,9 @@ func InitConfigurationData() error {
 		return err
 	}
 	runningConfig = config
+	if err := ReadRunModConfig(); err != false {
+		return fmt.Errorf("cannot read config")
```

Comment:
please use `log.Errorf` statement, and you should not return, rather continue the rest of the process